src/task3/DigitDifferencesFinder.java

```diff
@@ -0,0 +1,29 @@
+package task3;
+
+import java.util.Scanner;
+
+/* Дано любое натуральное 4-х значное число. Верно ли, что все цифры числа различны? */
+public class DigitDifferencesFinder {
+    public static void main(String[] args) {
+
+        Scanner input = new Scanner(System.in);
+        String number = input.nextLine();
+        boolean equivalenceIndicator = false;
+        for (int i = 0; i < number.length(); i++) {
+            if (equivalenceIndicator) {
```

Comment:
Could we change order of boolean expressions and do this method more readable? In this case I see than we always in first circle of our loop get false